### PR TITLE
83 frontend enhance picklist page

### LINKED ISSUE #83: Frontend: Enhance Picklist Page

### PR DESCRIPTION
**Pull request by qwertycloudhub**
Added teleOp Data

### CODE REVIEW

File: components/PicklistTable/index.tsx
Comment (drewbxyz) [inline components/PicklistTable/index.tsx:1]: @qwertycloudhub could you add the endgame columns as well?

File: app/picklist/page.tsx [hunk 1]
```diff
@@ -47,6 +47,7 @@ export default async function PickListPage() {
 
 	return (
 		<main className='mx-auto max-w-[540px]'>
```
Comment (drewbxyz) [inline app/picklist/page.tsx]: Add a bit of margin to the top, maybe mt-16?

**Review by drewbxyz — CHANGES_REQUESTED** [review]

File: components/PicklistTable/index.tsx
Comment (qwertycloudhub) [reply components/PicklistTable/index.tsx:1]: Got it

**Review by drewbxyz — APPROVED** [review]
lgtm

**Pull request merged**